### DISCUSSION TITLE
Question on https://de.accunit.access-codelib.net/Benutzeroberfl%C3%A4che

### DISCUSSION THREAD

**Discussion by FDominicus** [AccessCodeLib/AccUnit#29] [Q&A]
Yes, a gui interface in which one can see the tests and choose from wich to run, would be very appreciated.
IMHO testing should not be  pain, it should be easy, fast and accessible. I don't feel the output in the console is that.

**Comment by josef-poetzl** [comment]
I am currently doing well with the immediate window. :)

This is how it looked in the previous version:
<img src="http://en.accunit.access-codelib.net/images/1/15/AccUnit_VBE.png">

Concept for the VBE add-in would be to run standalone like the Access add-in and use the AccUnit library.

**Comment by FDominicus** [comment]
I see. It's quite clear to me that the screenshot is a nicer interface.
